pandas/tseries/tests/test_timezones.py

```diff
@@ -1398,6 +1398,44 @@ def test_normalize_tz(self):
         self.assertTrue(result.is_normalized)
         self.assertFalse(rng.is_normalized)
 
+    def test_normalize_tz_Asia(self):
+        import platform
+        if platform.system() == "Windows":
+            raise nose.SkipTest("timezone setting not supported on windows")
+
+        import os
+        import time
+
+        orig_tz = os.environ.get('TZ')
+
+        def setTZ(tz):
```

Comment:
ideally this would be setup as a context manager (and we put it in util.testing), then use it also in ``io/tests/test_pytables.py``